R/compute_kidney.R

```diff
@@ -139,7 +139,7 @@ compute_egfr <- function(creat, creatu = "SI", age, weight, sex, race = NULL, me
     optional = TRUE
   ))
   assert_numeric_vector(age)
-  assert_character_vector(sex, values = c("M", "F"))
+  assert_character_vector(sex)
```

Comment:
```suggestion
  assert_character_vector(sex, values = c("M", "F", NA_character_))
```
Then invalid values like `"Male"` are detected.